Lib/glyphsLib/builder/groups.py

```diff
@@ -87,13 +88,41 @@ def to_ufo_groups(self):
             source.font.groups[name] = glyphs[:]
 
 
+def _to_ufo_groups(glyph, groups, glyph_group, side, other_side):
```

Comment:
I suggest you call this method something like `_add_glyph_to_ufo_groups`; as it stands, it seems like it's making the whole ufo groups, whereas it's simply adding one glyph to some of them